docs/BR.md

```diff
@@ -1739,142 +1737,907 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+| \ \ \ \ _Effective 2021-10-01_    | MUST NOT     | -            | - |
+| `crlDistributionPoints`           | SHOULD NOT   | N            | See [Section 7.1.2.9.2](#71292-crl-distribution-points) |
+| Signed Certificate Timestamp List | MAY          | N            | See [Section 7.1.2.9.3](#71293-signed-certificate-timestamp-list) |
+| Any other extension               | SHOULD NOT   | -            | __**TBD**__ |
+
+##### 7.1.2.7.2 Authority Information Access
+
+For OCSP Responder certificates, this extension SHOULD NOT be present, as the Relying Party should already possess the necessary information. In order to validate the given Responder certificate, the Relying Party must have access to the Issuing CA's certificate, eliminating the need to provide `id-ad-caIssuers`. Similarly, because of the requirement for an OCSP Responder certificate to include the `id-pkix-ocsp-nocheck` extension, it is not necessary to provide `id-ad-ocsp`, as such responses will not be checked by Relying Parties.
+
+If present, the `AuthorityInformationAccesssSyntax` MUST contain one or more `AccessDescription`s. Each `AccessDescription` MUST only contain a permitted `accessMethod`, as detailed below, and each `AuthorityInformationAccessSyntax` MUST contain all required `AccessDescription`s.
+
+| __Access Method__ | __OID__            | __Presence__ | __Access Location__         | __Description__ |
+| --                | --                 | -            | ----                        | ---             |
+| `id-ad-caIssuers` | 1.3.6.1.5.5.7.48.2 | SHOULD NOT   | `uniformResourceIdentifier` | A HTTP URL of the Issuing CA's certificate. |
+| `id-ad-ocsp`      | 1.3.6.1.5.5.7.48.1 | MUST NOT     | `uniformResourceIdentifier` | A HTTP URL of the Issuing CA's OCSP responder. |
+| Any other value   | -                  | MUST NOT     | -                           | -               |
+
+##### 7.1.2.7.3 Basic Constraints
+
+| __Field__           | __Description__ |
+| ---                 | ------- |
+| `cA`                | MUST be FALSE |
+| `pathLenConstraint` | MUST NOT be present |
+
+**Note**: CAs MUST observe DER encoding rules, such as not explicitly encoding DEFAULT values within OPTIONAL fields.
+
+##### 7.1.2.7.4 Extended Key Usage
+
+| __Key Purpose__         | __OID__           | __Presence__ |
+| ----                    | ----              | -            |
+| `id-kp-OCSPSigning`     | 1.3.6.1.5.5.7.3.9 | MUST         |
+| `id-kp-serverAuth`      | 1.3.6.1.5.5.7.3.1 | MUST NOT     |
+| `id-kp-clientAuth`      | 1.3.6.1.5.5.7.3.2 | MUST NOT     |
+| `id-kp-codeSigning`     | 1.3.6.1.5.5.7.3.3 | MUST NOT     |
+| `id-kp-emailProtection` | 1.3.6.1.5.5.7.3.4 | MUST NOT     |
+| `id-kp-timeStamping`    | 1.3.6.1.5.5.7.3.8 | MUST NOT     |
+| `anyExtendedKeyUsage`   | 2.5.29.37.0       | MUST NOT     |
+| Any other value         | -                 | MUST NOT     |
```

Comment:
When we have a statement that says: "Any Other Value: MUST NOT", can we delete all the prior rows that also say MUST NOT since they are redundant?

Reply:
We could, but I tried to be explicit here on patterns where we've seen CAs have trouble, as well as to highlight the implications.

This is definitely an editorial strategy though, so I'm curious for more feedback. What I specifically want to avoid is any CA being able to say "The requirements were unclear", but I realize that's going to be subjective about which of those two approaches better achieves that.

Reply:
I went ahead and did this in the latest draft, but we can always undo this.